group_vars/all/ClusterConfiguration.yml

```diff
@@ -29,6 +29,7 @@ ClusterConfiguration:
   kubernetesVersion: "v{{ KUBERNETES_VERSION }}"
   # dns:
   networking:
+    cni_pod_name: "{{ POD_CNI_NAME }}"
```

Comment:
is there such a thing in the api?

Reply:
this is only used in your "grep" searches for pods. should i put it in another place in the variable stercture?